orchagent/pfcactionhandler.cpp

```diff
@@ -533,6 +535,22 @@ PfcWdZeroBufferHandler::~PfcWdZeroBufferHandler(void)
         SWSS_LOG_ERROR("Failed to set buffer profile ID on queue 0x%" PRIx64 ": %d", getQueue(), status);
         return;
     }
+
+    lockUnlockPriorityGroupAndQueue(portInstance, false);
+}
+
+void PfcWdZeroBufferHandler::lockUnlockPriorityGroupAndQueue(Port& port, bool lock) const
```

Comment:
>lockUnlockPriorityGroupAndQueue [](start = 29, length = 31)

The function name seems weird. I guess you mean 'setSomething'

Reply:
I agree. It seems that you use true to lock and false to unlock. IMO it is better to call the function setLock... and use true to lock and false to unlcok (the opposite of what you just did)

Reply:
Let me change to
```c++
void setPriorityGroupAndQueueLockFlag(Port& port, bool isLocked) const
```
As for "and use true to lock and false to unlcok" - this is exactly how it currently is.